### PR TITLE
screencapture: add page

### PR DESCRIPTION
**Pull request by dpinela**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/osx/screencapture.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# screencapture
+
+> Utility to take screenshots and screen recordings.
```
Comment (schneiderl) [inline pages/osx/screencapture.md]: It would be nice if we could find a link for this tool.

File: pages/osx/screencapture.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Take a screenshot of a selected rectangular area:
+
+`screencapture -i {{filename.png}}`
+
+- Take a screenshot after a delay:
+
+`screencapture -T {{seconds}} {{filename.png}}`
+
+- Make a screen recording:
```
Comment (schneiderl) [inline pages/osx/screencapture.md]:
```suggestion
- Make a screen recording and save it to a file:
```

Comment (schneiderl) [reply pages/osx/screencapture.md]: Just keeping it coherent with the first example.

File: pages/osx/screencapture.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# screencapture
+
+> Utility to take screenshots and screen recordings.
+
+- Take a screenshot and save it to a file:
+
+`screencapture {{filename.png}}`
```
Comment (schneiderl) [inline pages/osx/screencapture.md]:
```suggestion
`screencapture {{path/to/file.png}}`
```

File: pages/osx/screencapture.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+- Take a screenshot including the mouse cursor:
+
+`screencapture -C {{filename.png}}`
```
Comment (schneiderl) [inline pages/osx/screencapture.md]:
```suggestion
`screencapture -C {{path/to/file.png}}`
```

File: pages/osx/screencapture.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Take a screenshot and open it in Preview, instead of saving:
+
+`screencapture -P`
+
+- Take a screenshot of a selected rectangular area:
+
+`screencapture -i {{filename.png}}`
```
Comment (schneiderl) [inline pages/osx/screencapture.md]:
```suggestion
`screencapture -i {{path/to/file.png}}`
```

File: pages/osx/screencapture.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+- Take a screenshot after a delay:
+
+`screencapture -T {{seconds}} {{filename.png}}`
```
Comment (schneiderl) [inline pages/osx/screencapture.md]:
```suggestion
`screencapture -T {{seconds}} {{path/to/file.png}}`
```

File: pages/osx/screencapture.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+- Make a screen recording:
+
+`screencapture -v {{filename.mp4}}`
```
Comment (schneiderl) [inline pages/osx/screencapture.md]:
```suggestion
`screencapture -v {{path/to/file.mp4}}`
```

File: pages/osx/screencapture.md (hunk 1 above)
Comment (dpinela) [reply pages/osx/screencapture.md]: It's a builtin utility in macOS. I haven't been able to find the manpage on Apple's docs website, and none of the other builtins seems to have such a link, so I left it out.

Comment (schneiderl) [reply pages/osx/screencapture.md]: Oh. Ok then :)

**Comment by dpinela** [comment]
I've tweaked the examples as you suggested.

**Pull request merged**